simpleimage/make_rootfs.sh

```diff
@@ -17,10 +17,27 @@ DISTRO="$3"
 BOOT="$4"
 
 if [ -z "$DEST" -o -z "$LINUX" ]; then
-	echo "Usage: $0 <destination-folder> <linux-folder> [distro] [<boot-folder>]"
+	echo "Usage: $0 <destination-folder> <linux-folder-or-tarball> [distro] [boot-folder] [variables]"
+	echo ""
+	echo "Distros available:"
+	echo "arch - Arch Linux"
+	echo "xenial - Ubuntu Linux (Xenial Xerus)"
+	echo "sid - Debian Linux (sid)"
+	echo "jessie - Debian Linux (jessie)"
+	echo "opensuse - OpenSUSE Tumbleweed"
+	echo ""
+	echo "If no distro is specified, this tool will default to xenial"
+	echo ""
+	echo "Variables available:"
+	echo ""
+	echo "LCD=1       Enable LCD support"
```

Comment:
I do not see how this actually works. How is this variable set when passed as last argument to the script?